app/src/pages/inside/dashboardPage/dashboardList/dashboardTable/deleteColumn/deleteColumn.scss

```diff
@@ -0,0 +1,45 @@
+.cell {
+  position: relative;
+  display: table-cell;
+  padding: 15px;
+  vertical-align: top;
+  color: $COLOR--charcoal-grey;
+  font-size: 13px;
+  border-bottom: 1px solid $COLOR--gray-91;
```

Comment:
Please resolve problem with double borders for dashboard items in table view mode.